README.md

```diff
@@ -248,10 +248,9 @@ Contains details about an update package that has been downloaded locally or alr
 - __isFirstRun__: Flag indicating whether this is the first time the package has been run after being applied. (Boolean) This is useful for determining whether you would like to show a "What's New?" UI to the end-user after applying an update.
 
 ##### Methods
-- __apply(rollbackTimeout): Promise__: Applies this package to the application. The application will be reloaded with this package and on every application launch this package will be loaded.
-If the rollbackTimeout parameter is provided, the application will wait for a `notifyApplicationReady` for the given number of milliseconds.
-If `notifyApplicationReady` is called before the time period specified by rollbackTimeout, the apply operation is considered a success.
-Otherwise, the apply operation will be marked as failed, and the application is reverted to its previous version.
+- __apply(rollbackTimeout: Number = 0, restartImmediately: Boolean = true): Promise&lt;void&gt;__: Applies this package to the application, by unzipping its contents (e.g. the JS bundle) and saving it to the location on disk that the runtime expects to find the latest version of the app. If you pass a truthy value to the `restartImmediately` parameter, then after performing the apply, the app will be immediately restarted, so that the end-user sees the changes. Otherwise, the apply will complete, but they won't take affect until the next time that the app is started (i.e. the user starts your app after they or the OS killed it).<br /><br /> If a value greater then zero is provided to the `rollbackTimeout` parameter, the application will wait for the `notifyApplicationReady` method to be called for the given number of milliseconds.
```

Comment:
I want to just fix a few nittish things and rephrase slightly, is this okay?

- __apply(rollbackTimeout: Number = 0, restartImmediately: Boolean = true): Promise&lt;void&gt;__: Applies this package to the application by unzipping its contents (e.g. the JS bundle) and saving it to the location on disk where the runtime expects to find the latest version of the app. If the `restartImmediately` parameter is set to false, the apply will complete, but it won't take effect until the next time that the app is restarted. Otherwise, the app will be immediately restarted after performing the apply, so that the end-user sees the changes.<br /><br /> If a value greater than zero is provided to the `rollbackTimeout` parameter, the application will wait for the `notifyApplicationReady` method to be called for the given number of milliseconds.

Also, again a little nittish, but is it a bit strange that we are talking about 'restartImmediately' before we talk about 'rollbackTimeout', considering the arguments are the other way around?